src/build.js

```diff
@@ -70,8 +83,21 @@ const followBuild = async ({ appSlug, buildSlug, client }, options = {}) => {
     }
 
     // If the build is completed and there are no more log chunks
-    if (!!attributes.build_finished && response.data.log_chunks.length === 0) {
-      // No need to keep watching a completed build
+    if (attributes.finished_at) {
+      if (response.data.log_chunks.length === 0) {
+        // No need to keep watching a completed build
+        break;
+      }
+
+      // If the build is done and chuncks are being seeing, then pull them
+      // as fast as possible
+      interval = 0;
+    }
+
+    // If the log was being followed and it switched to streaming before the
```

Comment:
Another typo. "streaming" should be "archived" I'll follow up with a fix for both